packages/git/src/browser/blame/blame-contribution.ts

```diff
@@ -131,22 +136,47 @@ export class BlameContribution implements FrontendApplicationContribution, Comma
 
     registerMenus(menus: MenuModelRegistry): void {
         menus.registerMenuAction(EDITOR_CONTEXT_MENU_GIT, {
-            commandId: BlameCommands.SHOW_GIT_ANNOTATIONS.id,
-            label: BlameCommands.SHOW_GIT_ANNOTATIONS.label!.slice('Git: '.length)
+            commandId: BlameCommands.TOGGLE_GIT_ANNOTATIONS.id,
+            label: BlameCommands.TOGGLE_GIT_ANNOTATIONS.label!.slice('Git: '.length)
         });
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
         keybindings.registerKeybinding({
-            command: BlameCommands.SHOW_GIT_ANNOTATIONS.id,
+            command: BlameCommands.TOGGLE_GIT_ANNOTATIONS.id,
             context: EditorKeybindingContexts.editorTextFocus,
             keybinding: 'alt+b'
         });
         keybindings.registerKeybinding({
             command: BlameCommands.CLEAR_GIT_ANNOTATIONS.id,
-            context: EditorKeybindingContexts.strictEditorTextFocus,
+            context: BlameAnnotationsKeybindingContext.showsBlameAnnotations,
             keybinding: 'esc'
         });
     }
 
 }
+
+@injectable()
+export class BlameAnnotationsKeybindingContext implements KeybindingContext {
```

Comment:
Should not it extend `EditorTextFocusContext` similar to `JavaEditorTextFocusContext` that it is active only when an editor has focus?

Reply:
thought about that indeed. let me check if that works.

Reply:
works nicely, thanks @akosyakov!